packages/cli/lib/services/docs.service.ts

```diff
@@ -137,7 +150,7 @@ function generalInfo(scriptPath: string, endpointType: string, scriptConfig: Nan
         console.warn(`Warning: no description for ${scriptPath}`);
     }
 
-    const endpoints = Array.isArray(scriptConfig.endpoint) ? scriptConfig.endpoint : [scriptConfig.endpoint];
+    const endpoints = scriptConfig.type === 'action' ? [scriptConfig.endpoint] : scriptConfig.endpoints;
     const [endpoint] = endpoints;
```

Comment:
not sure why this is not a oneliner and why not keep the isArray check like before?

Reply:
Types are different, scriptConfig.endpoint only exists in action, in sync it's plural.